src/pages/Signup.jsx

```diff
@@ -11,14 +11,12 @@ import Box from '@mui/material/Box';
 import Container from '@mui/material/Container';
 import Paper from '@mui/material/Paper';
 
-// Custom Hook Imports
-import useContactsList from '@hooks/useContactsList';
-import useSession from '@hooks/useSession';
 // Constant Imports
 import { ENV } from '@constants';
 
 // Signup Form Imports
 import { PodRegistrationForm, ShowNewPod, initializePod, registerPod } from '@components/Signup';
+import { Typography } from '@mui/material';
```

Comment:
Very minor nitpick similar to the previous comment about MUI imports and consistency.